packages/db/migrations/mysql/0000_harsh_photon.sql

```diff
@@ -1,8 +1,8 @@
 CREATE TABLE `account` (
-	`userId` varchar(256) NOT NULL,
+	`userId` varchar(64) NOT NULL,
 	`type` text NOT NULL,
-	`provider` varchar(256) NOT NULL,
-	`providerAccountId` varchar(256) NOT NULL,
+	`provider` varchar(64) NOT NULL,
+	`providerAccountId` varchar(64) NOT NULL,
```

Comment:
Why go from 256 to 64 ?

Reply:
I went down to 64 as it needs a lot less space and we actually do not need 256 characters. Additionally there were some issues because some keys were defined with text and so the indexing of it wasn't really possible

Reply:
Alright makes sense